index.md

```diff
@@ -13,29 +13,27 @@ nav_order: 0
 
 This is the `karaoke!`'s offical site.
 
-This ruleset is based on `osu!`, and it's a part of un-official osu ruleset(gamemode).
+This project is an un-official osu lazer ruleset.
 
-It's still in wip, anything news will be update in here.
+It is still a work in progress, and new information will be posted here.
 
-## What's the feature in this ruleset?
+## What are this rulesets current features?
 
-- Base karaoke feature, display lyric with simple animation.
-- Song selection.
-- Saiten(scoring) system.
-- Support japanese.
-- Cross platform, you can run it in pc and mobile device.
+- Basic karaoke gameplay, with a simply animated lyric overlay.
+- Custom songs made for the karaoke ruleset (no conversion of beatmaps yet).
+- A Saiten(scoring) system.
+- Japanese support.
+- Cross platform, pc and mobile are both supported.
 
-## What's the feature other karaoke software don't have 
+## What are the exclusive features of this ruleset?
 
 - More complex lyric animation.
-- Will support most language.
-- Karoake beatmap with translate support.
-- Romaji support so you can sing the song all over the world.
+- Will support most languages.
+- Karoake beatmaps with translation support.
+- Romaji support so japanese songs are more accessible.
```

Comment:
Might not japanese song only 🤔 (Although japanese song is popular).

Reply:
this part is ok because it means that it's not only Japanese but you can do it even without knowing hiragana because romaji is supported